src/infrastructure/databases.md

```diff
@@ -110,7 +110,7 @@ statement afterwards. The statements are also executed after the index rebuild h
 
 ### Keep indexes in memory
 
-If you have enough RAM, you should try to keep all indexes in memory to reduce the need to access the hard disc to a minimum. Nevertheless, make sure that key_buffer_size + innodb_buffer_pool_size does not exceed 75% of the available RAM and that there's enough RAM for the database connections and the operating system.
+If you have enough RAM, you should try to keep all indexes in memory to reduce the need to access the hard disc to a minimum. Nevertheless, make sure that `key_buffer_size` and `innodb_buffer_pool_size` do not exceed 75% of the available RAM and that there is enough RAM for the database connections and the operating system.
```

Comment:
`key_buffer_size + innodb_buffer_pool_size` is intended because the sum should not exceed 75% RAM